templates/boxMenuItem.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Adapt from 'core/js/adapt';
+import { classes, compile } from 'core/js/reactHelpers';
+
+export default function BoxMenuItem (props) {
+
+  const {
+    displayTitle,
+    _graphic,
+    body,
+    duration,
+    linkText,
+    _isVisited,
+    _isLocked,
+    _isComplete,
+    title,
+    _isOptional,
+    _nthChild,
+    _totalChild
+  } = props;
+
+  const _globals = Adapt.course.get('_globals');
+
+  const durationLabel = duration
+    ? [_globals?._menu?._boxMenu?.durationLabel.trim(), duration].filter(Boolean).join(' ')
+    : '';
+
+  const visited = _isVisited ? _globals?._accessibility?._ariaLabels?.visited : '';
+  const complete = _isComplete ? _globals?._accessibility?._ariaLabels?.complete : '';
+  const completion = complete || visited;
+  const locked = _isLocked ? _globals?._accessibility?._ariaLabels?.locked : linkText;
+  const optional = _isOptional ? _globals?._accessibility?._ariaLabels?.optional : '';
+  const itemCount = compile(_globals?._menu?._boxMenu?.itemCount || '', { _nthChild, _totalChild });
+  const ariaLabel = [
+    completion, locked, `${title}.`, `${itemCount}.`, `${optional}`
+  ].filter(Boolean).join(' ');
+
+  return (
+    <div className="menu-item__inner boxmenu-item__inner">
+
+      {_graphic?.src &&
+      <div className="menu-item__image-container boxmenu-item__image-container">
+        <img className="menu-item__image boxmenu-item__image" src={_graphic.src} aria-hidden="true" />
```

Comment:
Perhaps splitting the attributes across multiple lines helps with readability? 
```suggestion
        <img 
          className="menu-item__image boxmenu-item__image" 
          src={_graphic.src} 
          aria-hidden="true" 
        />
```